<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeacherManagement</class>
 <widget class="QWidget" name="TeacherManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>教师模块</string>
  </property>
  <widget class="QTableView" name="mainTable">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>841</width>
     <height>491</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="willSort">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>40</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>按分数排序</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>分析</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>40</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>筛选成绩</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="minScore">
      <property name="placeholderText">
       <string>最低分</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>到</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="maxScore">
      <property name="placeholderText">
       <string>最高分</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>451</x>
     <y>10</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>搜索班级:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="classSearch">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>411</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>搜索学生:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="studentSearch">
      <property name="placeholderText">
       <string>学号或姓名</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>411</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>搜索教师</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="teacherSearch"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>40</y>
     <width>401</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>搜索课程:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="courseSearch">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
